charts/lagoon-core/templates/api-db.statefulset.yaml

```diff
@@ -13,6 +13,10 @@ spec:
     metadata:
       annotations:
         checksum/api-db.secret: {{ include (print $.Template.BasePath "/api-db.secret.yaml") . | sha256sum }}
+        {{- if .Values.k8up }}
+        appuio.ch/backupcommand: /bin/sh -c "mysqldump --max-allowed-packet=500M --events --routines --quick --add-locks --no-autocommit --single-transaction --all-databases"
```

Comment:
let's use the default annotations `k8up.syn.tools/backupcommand` and `k8up.syn.tools/file-extension` of k8up, see https://k8up.io/k8up/1.1.0/references/config-reference.html

Reply:
Good catch; I just copypasta'd those a bit too quickly